### PR TITLE
Positions

### PR DESCRIPTION
**Pull request by nabucopena**
At `/positions`, we show a table with the amount of games played / games won by each player.

Screenshot 2020-05-28 at 14 55 03

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -118,6 +125,14 @@ def conn
 
   on get do
 
+    on "positions" do
+      on !session[:user] do
```
Comment (carlosipe) [inline app.rb:129]: `on !current_user ?`

File: views/positions.mote [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <th>Player</th>
+        <th>Played</th>
+        <th>Won</th>
+      </tr>
+    </thead>
+    <tbody>
+      % players.each do |player|
+         <tr>
+            <td>{{player.fetch("name")}}</td>
+            <td>{{this.get_score(player.fetch("id"))[0]}}</td>
```
Comment (carlosipe) [inline views/positions.mote:14]:
```html
<td>player.fetch("played_amount")</td>
<td>player.fetch("won_amount")</td>
```

Comment (carlosipe) [reply views/positions.mote:14]: We need players to be like:
```rb
[
  { "name"=> "Félix",
    "played_amount" => 5,
    "won_amount" => 3
  },
  {
    ...
  }
]
```

File: app.rb (hunk 1 above)
Comment (nabucopena) [reply app.rb:122]: i have to fix that in several files, in another PR i'll change it

File: views/positions.mote (hunk 2 above)
Comment (nabucopena) [reply views/positions.mote]: i made something similar to that

Comment (nabucopena) [reply views/positions.mote]: i have changed that code, now the array "players" includes the played/won amount

File: app.rb [hunk 3]
```diff
@@ -118,6 +118,22 @@ def conn
...
+        res.redirect("/login")
+      end
+      sql = <<~SQL
+        select name,
+        count(*) filter(where id_player_1 is not null) as played,
+        count(*) filter(where (id=id_player_1 and score_1>score_2) or (id=id_player_2 and score_2>score_1)) as won
+        from players
+        left join games on id_player_1=id or id_player_2=id
```
Comment (carlosipe) [inline app.rb:130]: Use the `my_games` view?

**Pull request merged**